client/src/Navigator.ts

```diff
@@ -14,14 +14,22 @@ import {
     RoutesScreen,
 } from './screens';
 
-import {Routes} from './consts'
+import {Routes} from './consts';
 
 const Tabs = createBottomTabNavigator({
     [Routes.MAP]: MapScreen,
     [Routes.POI]: PoiScreen,
     [Routes.ROUTES]: RoutesScreen,
     [Routes.FAVORITES]: FavoritesScreen,
 }, {
+    tabBarOptions: {
+        showLabel: false,
+        activeTintColor: '#eb5757',
+        inactiveTintColor: '#666',
+        style: {
+            height: 60,
```

Comment:
Don't you think it might be large for smaller devices?

Reply:
it might. currently i only look at it on my android phone.

Reply:
ok then, we can align these things after mvp 😄